src/BenchmarksApps/Mvc/benchmarks.certapi.yml

```diff
@@ -0,0 +1,210 @@
+﻿# Examples:
+# --config benchmarks.certapi.yml --scenario ApiCrudListProducts --profile aspnet-perf-lin
+
+imports:
+- https://raw.githubusercontent.com/dotnet/crank/haok/crank/src/Microsoft.Crank.Jobs.HttpClient/httpclient.yml
```

Comment:
update branch to master everywhere in this file before merge

Reply:
Worst case we can merge and do the change after (before the branch is closed)